test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -184,6 +262,53 @@ public void windowClosing(WindowEvent e) {
         windowList.add(frame);
     }
 
+    private static void captureScreen(Rectangle bounds) {
+        if (robot == null) {
+            try {
+                robot = new Robot();
+            } catch (AWTException e) {
+                String errorMsg = "Failed to create the " +
+                        "instance of Robot.";
```

Comment:
```suggestion
                String errorMsg = "Failed to create an " +
                        "instance of Robot.";
```
Maybe make it a constant?